src/containers/BuySellCrypto/PaymentMethod/ManageWyreAccount/ManageWyrePersonalDetails.js

```diff
@@ -32,9 +36,29 @@ class ManageWyrePersonalDetails extends Component {
         dateOfBirth: null,
         socialSecurityNumber: null,
       },
+      date: new Date().toISOString(),
+      mode: 'date',
+      toggleCalendar: false,
     };
   }
 
+  toggleCalendar = () => {
+    this.setState({
+      // toggleCalendar: Platform.OS === 'ios' ? true : false,
+      toggleCalendar: !this.state.toggleCalendar
```

Comment:
If your new state update depends on the previous state, always use the functional form of setState which accepts as argument a function that returns a new state. See [https://reactjs.org/docs/state-and-lifecycle.html#state-updates-may-be-asynchronous](url)
this.setState(prevState => ({
  toggleCalendar: !prevState. toggleCalendar
}));